Lib/zoneinfo/_tzpath.py

```diff
@@ -19,16 +19,25 @@ def reset_tzpath(to=None):
     else:
         env_var = os.environ.get("PYTHONTZPATH", None)
         if env_var is not None:
-            base_tzpath = _parse_python_tzpath(env_var)
+            base_tzpath = _parse_python_tzpath(env_var, stacklevel)
         else:
             base_tzpath = _parse_python_tzpath(
-                sysconfig.get_config_var("TZPATH")
+                sysconfig.get_config_var("TZPATH"),
+                stacklevel,
             )
```

Comment:
Since the repeated code is now more complex, I suggest to refactor it:
```suggestion
        if env_var is None:
            env_var = sysconfig.get_config_var("TZPATH")
        base_tzpath = _parse_python_tzpath(env_var, stacklevel)
```

Reply:
<img width="747" alt="Снимок экрана 2024-02-05 в 14 11 11" src="https://github.com/python/cpython/assets/4660275/cf6984ab-9c07-4b66-954f-5c36f60d5be9">
I will do that manually 👍